demo/demo-styles.css

```diff
@@ -0,0 +1,20 @@
+*{
+	transition: ease-in-out 150ms;
+}
+
+.col {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: sans-serif;
+  color: white; 
+  min-height: 150px;
```

Comment:
Out of curiosity: Why `150px`?

Reply:
Demo purpose only, makes the cols have a comfortable amount of height